stream_alert_cli/terraform/threat_intel_downloader.py

```diff
@@ -53,6 +53,7 @@ def generate_threat_intel_downloader(config):
         'table_wcu': ti_downloader_config.get('table_wcu', '10'),
         'ioc_keys': ti_downloader_config.get('ioc_keys'),
         'ioc_filters': ti_downloader_config.get('ioc_filters'),
-        'ioc_types': ti_downloader_config.get('ioc_types')
+        'ioc_types': ti_downloader_config.get('ioc_types'),
```

Comment:
do these need default values?

Reply:
Yes, the default values are set in the [argparser](https://github.com/airbnb/streamalert/blob/master/manage.py#L1372-L1376)

Reply:
But that doesn't mean somebody can't modify the config by hand

Reply:
Aha, good point! I will add default values.